src/NzbDrone.Core/IndexerSearch/Definitions/AlbumSearchCriteria.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace NzbDrone.Core.IndexerSearch.Definitions
+{
+    public class AlbumSearchCriteria : SearchCriteriaBase
+    {
+        public override string ToString()
+        {
+            return $"[{Album.Title}]";
```

Comment:
That works as long as it's to too specific, do most releases have year in release title somewhere?